src/main/resources/static/assets/css/custom.css

```diff
@@ -53,4 +53,17 @@
 .vertical-align {
     align-items: center;
     display: flex;
-}
\ No newline at end of file
+}
+
+.rd-navbar-nav > li > a.slack {
```

Comment:
```.rd-navbar-nav > li > a ```
has color #fff and to change that color (from selector in custom.css) I saw 2 options to make it:
- use !important (I wanted to avoid)
- create that (a bit) complicated path

Reply:
Second option is better

Reply:
Ok, so I won't change that.